python/tests/test_regression.py

```diff
@@ -0,0 +1,62 @@
+from vw_executor.vw import Vw
+from vw_executor.vw_opts import Grid
+from numpy.testing import assert_allclose
+import pandas as pd
+import numpy as np
+import random
+
+with open("regression1.txt", "w") as f:
+    for i in range(10000):
+        x = random.uniform(1, 100)
+        y = 5
+        f.write(f"{y} |f x:{x}\n")
+
+
+def core_test(files, grid, outputs, job_assert):
+    vw = Vw(".vw_cache", reset=True)
```

Comment:
probably better to add handler=None. I think now it is trying to draw progress bars.